src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -267,6 +357,136 @@ function isCtrlKey(e) {
         }
     });
 
+    if (doc.evaluate) {
```

Comment:
you also need to check for ``Array.from``, as you rely on it, and not all browsers have it